src/components/Books.js

```diff
@@ -0,0 +1,20 @@
+import { React } from 'react';
+import Book from './Book';
+import Form from './Forms';
+
+function Books() {
+  return (
+    <div>
+      <h2>Books</h2>
+      <ul>
+        <li>
+          <Book />
+          <div><button type="button">Remove</button></div>
```

Comment:
- [ ] I believe it would be more logical to put this `remove` button inside the **`Book`** component. That way it would be easier to tie the removal action to a book. Kindly do that.